etc/hub.api.md

```diff
@@ -750,7 +750,9 @@ export interface Period {
 }
 
 // @public
-export type Private = Identity;
+export interface Private extends Identity {
+    bytes: Uint8Array;

```

Comment:
this is a breaking interface change... there doesn't appear to be any consequences in our own codebase, but it should lead to a minor bump (we're adding a value). Check me on this though pls.